src/app/modules/myinfo/myinfo.service.ts

```diff
@@ -457,13 +459,13 @@ export class MyInfoServiceClass {
   /**
    * Decodes and verifies a JWT from MyInfo containing the user's
    * UIN/FIN.
-   * @param accessToken Access token JWT
+   * @param loginJwt Login JWT
    */
   extractUinFin(
-    accessToken: string,
+    loginJwt: string,
   ): Result<string, MyInfoInvalidAccessTokenError> {
     return Result.fromThrowable(
-      () => this.#myInfoGovClient.extractUinFin(accessToken),
+      () => jwt.verify(loginJwt, spcpMyInfoConfig.myInfoJwtSecret),
       (error) => {
         logger.error({
           message: 'Error while extracting uinFin from MyInfo access token',
```

Comment:
suggest - update log msg to jwt has expired?

Reply:
hmm, it's not necessarily expiry here, e.g. if the client submits arbitrary garbage for JWT, it just can't be decoded. but I've fixed the log message above on line 418 (from "access token" to "login JWT")